### DISCUSSION TITLE
Increase storage on search node.

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#12934] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

How do you increase storage (/nsm) on a storage node. 

I can expand the virtual drive how do I give it to the system?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
Please see the documentation:
https://docs.securityonion.net/en/2.4/new-disk.html
